cmd/up/controlplane/controlplane.go

```diff
@@ -52,6 +51,12 @@ func (c *Cmd) AfterApply(kongCtx *kong.Context) error {
 	}
 	kongCtx.Bind(upCtx)
 
+	client, err := upCtx.BuildCurrentContextClient()
+	if err != nil {
+		return errors.Wrap(err, "unable to get kube client")
+	}
+	kongCtx.Bind(client)
```

Comment:
kong is unable to bind interfaces via `Bind()` , instead use `BindTo()`
https://github.com/alecthomas/kong/issues/48#issuecomment-583695243

```suggestion
	kongCtx.BindTo(client, (*ctrlclient.Client)(nil))
```

Reply:
Fixed in https://github.com/RedbackThomson/up/pull/1